prober/http.go

```diff
@@ -247,6 +333,26 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 	if resp == nil {
 		resp = &http.Response{}
 	}
+	i := 0
+	for _, trace := range tt.traces {
+		// We get the duration for the first request from chooseProtocol
+		if i != 0 {
+			durationGaugeVec.WithLabelValues("resolve").Add(trace.dnsDone.Sub(trace.start).Seconds())
+		}
+		if resp.TLS != nil {
```

Comment:
Just because the last response wasn't TLS doesn't mean previous ones were, you always should add in tls

Reply:
Oh good catch. I think we need to track which trace involved TLS.